boards/v2/qemu/qemu_xtensa/Kconfig.defconfig

```
@@ -0,0 +1,12 @@
+# Copyright (c) 2017, 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_QEMU_XTENSA || BOARD_QEMU_XTENSA_DC233C_MMU
```

`BOARD_QEMU_XTENSA_DC233C_MMU` is redundant in this case.
```suggestion
if BOARD_QEMU_XTENSA
```

---

Fixed